assets/js/previews/products.js

```diff
@@ -15,6 +15,14 @@ export const previewProducts = [
 		variation: '',
 		permalink: 'https://example.org',
 		sku: 'wp-pennant',
+		summary: __(
+			'Fly your WordPress banner with this beauty! Deck out your office space or add it to your kids walls. This banner will spruce up any space it’s hung!',
+			'woo-gutenberg-products-block'
+		),
+		short_description: __(
+			'Fly your WordPress banner with this beauty! Deck out your office space or add it to your kids walls. This banner will spruce up any space it’s hung!',
+			'woo-gutenberg-products-block'
+		),
 		description: __(
 			'Fly your WordPress banner with this beauty! Deck out your office space or add it to your kids walls. This banner will spruce up any space it’s hung!',
 			'woo-gutenberg-products-block'
```

Comment:
What about defining this content once and then just referencing it for each property?  That way if we need to update this text in the future for some reason there's only one place to update?

Reply:
Good idea. I also added some generic description text to all so it's different to the short description, in case someone tests with the wrong field.